go/fiber/go.mod

```diff
@@ -0,0 +1,12 @@
+module main
```

Comment:
could you reduce this file using 
```
module main

require github.com/fenny/fiber v0.4.0
```
(without specifying `go` **version**)

it reduce maintenance stuff :stuck_out_tongue: